tls/_constructs.py

```diff
@@ -19,24 +20,29 @@
     "TLSPlaintext",
     UBInt8("type"),
     ProtocolVersion,
-    UBInt16("length"),  # TODO: Reject packets with length > 2 ** 14
-    Bytes("fragment", lambda ctx: ctx.length),
+    PrefixedBytes("fragment", UBInt16("length")),  # TODO: Reject
+                                                   # packets with
```

Comment:
Mildly offtopic note - we probably should start thinking about the TODO marked here before we get too far in the implementation to come back and address this.